edb/testbase/server.py

```diff
@@ -2451,6 +2451,32 @@ async def __aenter__(self):
                 timeout=240,
             ),
         )
+
+        output = b''
+
+        async def read_stdout():
+            nonlocal output
+            # Tee the log temporarily to a tempfile that exists as long as the
+            # test is running. This helps debug hanging tests.
+            with tempfile.NamedTemporaryFile(mode='w+t',
+                                             prefix='edgedb-test-log-',
+                                             delete=False) as temp_file:
```

Comment:
Why `delete=False` and a manual `remove`?

Reply:
Ack, that was an oversight, will fix before landing.